pkg/scheduler/internal/cache/cache_test.go

```diff
@@ -1239,6 +1241,32 @@ func TestSchedulerCache_UpdateSnapshot(t *testing.T) {
 		podsWithAffinity = append(podsWithAffinity, pod)
 	}
 
+	// Add a few of pods with PVC, starting on the third node.
+	var podsWithPVC []*v1.Pod
+	for i := 2; i < 8; i++ {
```

Comment:
to distiinguish with podWithAffinity, you can build them in a declarative manner:

```
podsWithAffinity := []*v1.Pod{
    st.MakePod().Name("p-affinity-0").Node("node0")...,
}

podsWithPVC := []*v1.Pod{
    st.MakePod().Name("p-pvc-0").Node("node0")...,
    ....
}
```

Then we don't need to start with index 2, which makes the test hard to read.

Reply:
Thanks, done!